src/main/java/camp/woowak/lab/menu/repository/MenuRepository.java

```diff
@@ -1,8 +1,14 @@
 package camp.woowak.lab.menu.repository;
 
+import java.util.Optional;
+
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 
 import camp.woowak.lab.menu.domain.Menu;
 
 public interface MenuRepository extends JpaRepository<Menu, Long> {
+	@Query("SELECT m from Menu m join fetch Store s on m.store = s where m.id = :id")
```

Comment:
```suggestion
	@Query("SELECT m FROM Menu m JOIN FETCH m.store WHERE m.id = :id")
```
위와 같이 수정해도 똑같은 결과가 나오고 JPA 관계 매핑을 좀 더 잘 사용할 수 있는 방법이라는데 한번 테스트 가능하신가요 ?

적용하지 않으셔도 됩니다! 간단한 제안이에요 :)

Reply:
같은 결과를 내고 더 깔끔해보여서 바로 적용했습니다! 감사합니다 :)